db/mercenary_skill_db.txt

```diff
@@ -215,3 +215,5 @@
 2347,8202,5	//MS_MAGNUM
 2347,8226,1	//MER_REGAIN
 2347,8233,1	//MER_AUTOBERSERK
+// M_LOKI
+2937,8241,1	//MER_INVINCIBLEOFF2
```

Comment:
after recompile got this error

![image](https://user-images.githubusercontent.com/29746866/39417117-c15ba6c2-4c84-11e8-8dfe-8b96c15e028b.png)

Reply:
As you can see below all builds checks have passed. Do you happen to merge manually (copy-past) ? If you do please use github tool instead, it would be better for you and me. If not please ignore my comment.